services/traction/api/jobs/endorser_connection_job.py

```diff
@@ -8,6 +8,8 @@
 class EndorserConnectionJob(Job):
     def __init__(self, profile: Profile):
         super().__init__(profile, TenantJobType.endorser)
+        self.state_handlers[TenantJobStatusType.processing.lower()] = self.on_processing
+        self.state_handlers[TenantJobStatusType.completed.lower()] = self.on_completed
```

Comment:
should we change the enum values to be lower case then use .capitalize when we need to make it 'pretty'?

Reply:
it's probably best to follow what acapy presents for values. hopefully it is consistent. the bigger part would be how we send them back to the tenant, like are we going to use resources to have a display? bigger talk, but lower is visually more appealing to me as a value, UPPER seems better for the enum names - imo.

Reply:
i guess since this is all new and no other dependencies or API using this, i can just make the values lower... let me do that and this will be cleaner.

Reply:
i just figure the enums should be 'for the code first' then 'made pretty' when being shown to users. 

If someone misses a 'lower()` or `upper()` I'd rather it be a 'minimal' bug like a label not being capitalised, then a hard to track down bug because a webhook handler isn't matching the enum right. 

I don't know what's more common.

Reply:
just log these questions, either in JIRA or somewhere. these are legitimate questions and should inform our design going forward.